puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-test.yml

```diff
@@ -0,0 +1,102 @@
+---
+- job-template:
+    name: foreman-{version}-{distro}-{action}-test
+    project-type: 'pipeline'
+    sandbox: true
+    parameters:
+      - string:
+          name: expected_version
+          default: 'UNKNOWN'
+      - string:
+          name: version
+          default: '{version}'
+      - string:
+          name: distro
+          default: '{distro}'
+      - string:
+          name: action
+          default: '{action}'
+      - string:
+          name: type
+          default: 'foreman'
+    dsl:
+      !include-raw:
+        - pipelines/lib/foremanCentosJob.groovy{empty}
+        - pipelines/lib/pipelines.groovy{empty}
+        - ../theforeman.org/pipelines/lib/duffy.groovy{empty}
+        - ../theforeman.org/pipelines/lib/ansible.groovy{empty}
+        - ../theforeman.org/pipelines/lib/foreman_infra.groovy{empty}
+
+- project:
+    name: foreman-release-test-1.23
```

Comment:
1.23 is no longer supported now that 2.0 is GA so can be dropped. Perhaps that we should do that first in master?